### PR TITLE
docs: Add compression related docs

### PR DESCRIPTION
**Pull request by Xuanwo**
Signed-off-by: Xuanwo <github@xuanwo.io>

I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

docs: Add compression related docs

## Changelog

- Documentation

### CODE REVIEW

File: docs/doc/21-load-data/01-s3.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ description:
 <img src="https://datafuse-1253727613.cos.ap-hongkong.myqcloud.com/load/load-data-from-s3.png" width="550"/>
 </p>
 
-Databend [COPY](../30-reference/30-sql/10-dml/dml-copy.md) command can read object files(CSV or Parquet file format) from Amazon S3 buckets.
+Databend [COPY](../30-reference/30-sql/10-dml/dml-copy-into-table.md) command can read object files(CSV or Parquet file format) from Amazon S3 buckets.
```
Comment (BohuTANG) [inline docs/doc/21-load-data/01-s3.md:12]: We also support `json` file format, which can be added here by the way?

Comment (Xuanwo) [reply docs/doc/21-load-data/01-s3.md:12]: Seems we don't have the json sample data?

```shell
> curl https://datafuse-1253727613.cos.ap-hongkong.myqcloud.com/data/books.parquet
<?xml version='1.0' encoding='utf-8' ?>
<Error>
        <Code>AccessDenied</Code>
        <Message>Access Denied.</Message>
        <Resource>datafuse-1253727613.cos.ap-hongkong.myqcloud.com/data/books.json</Resource>
        <RequestId>NjI5NDI4ZDhfZTFhNzM4MDlfYzJjY18yMjFkNDJl</RequestId>
        <TraceId>OGVmYzZiMmQzYjA2OWNhODk0NTRkMTBiOWVmMDAxODc0OWRkZjk0ZDM1NmI1M2E2MTRlY2MzZDhmNmI5MWI1OTVmYzc0YmEzZDg5YTVjMDA4YzA3ZWY5ZjQwZTE2ZGU1MTE5N2Q3ZmIwN2M0MWZlZWViZTBmZTIwMzc4M2U4NjU=</TraceId>
</Error>
```

**Comment by Xuanwo** [comment]
Updated for https://github.com/datafuselabs/databend/pull/5669

**Review by BohuTANG — APPROVED** [review]
:100:

**Pull request merged**